lib/util/Arrays.v3

```diff
@@ -3,53 +3,62 @@
 
 // Utility methods for arrays.
 component Arrays {
-	// get the length of {array}, but returning 0 for a null array
```

Comment:
Please move lib/util changes to a separate PR.

Reply:
Do you want the `src/util/Arrays.v3` and `src/util/Strings.v3` changes in that PR as well?

Reply:
I've currently just reverted `lib/util/Arrays.v3` to its original state. If you want the above two in a different PR as well, then I think it'll be easier (for both of us) to close this PR and make new ones.